karspexet/venue/management/commands/build_seats.py

```diff
@@ -0,0 +1,34 @@
+from django.core.management.base import BaseCommand, CommandError
+from karspexet.venue.models import Venue, Seat, SeatingGroup
+
+import csv
+
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        parser.add_argument("venue_id", type=int)
+
+        parser.add_argument("file")
+
+    def handle(self, *args, **options):
+        venue = Venue.objects.get(pk=options["venue_id"])
+
+        with open(options["file"]) as csv_file:
+            reader = csv.reader(csv_file, delimiter="\t")
+
+            groupings = {}
+
+            for row in reader:
```

Comment:
Why not use some destructuring here? like `for row, _ in reader` or perhaps `for head, *tail in reader`.

Reply:
Because I didn't know that was a thing. Thanks for pointing it out!